src/common/helpers/index.js

```diff
@@ -23,6 +23,7 @@ import {
     addressFormValidation
 } from "./FormValidation";
 import { useVideoActions } from "./VideoActions";
+import { useSearchVideos } from "./SearchVideosHandler";
```

Comment:
If you want to avoid doing `.` while importing then you can look into absolute paths.